setup.py

```diff
@@ -43,6 +43,7 @@
         'flask==1.1.1',
         'flask-restplus==0.13.0',
         'GitPython==3.0.3',
+        'pymongo[tls]==3.8.0',
```

Comment:
If you are planning on using a `mongodb+src` url (which I think atlas uses), you'll need to install [`dnspython`](https://github.com/mongodb/mongo-python-driver#dependencies)